libs/gi/page-team/src/TeamSetting/TeamExportModal.tsx

```diff
@@ -0,0 +1,315 @@
+import {
+  BootstrapTooltip,
+  CardThemed,
+  ModalWrapper,
+  SqBadge,
+} from '@genshin-optimizer/common/ui'
+import { toggleInArr } from '@genshin-optimizer/common/util'
+import {
+  defLoadoutExportSetting,
+  type LoadoutDataExportSetting,
+  type LoadoutDatum,
+  type LoadoutExportSetting,
+} from '@genshin-optimizer/gi/db'
+import { useDatabase, useTeam, useTeamChar } from '@genshin-optimizer/gi/db-ui'
+import {
+  BuildIcon,
+  CharIconSide,
+  CustomMultiTargetIcon,
+  FieldDisplayList,
+} from '@genshin-optimizer/gi/ui'
+import CloseIcon from '@mui/icons-material/Close'
+import InfoIcon from '@mui/icons-material/Info'
+import {
+  Alert,
+  Box,
+  Button,
+  CardContent,
+  CardHeader,
+  Checkbox,
+  Divider,
+  Grid,
+  IconButton,
+  ListItem,
+  ListItemButton,
+  ListItemText,
+  Typography,
+} from '@mui/material'
+import { useState } from 'react'
+
+// TODO: Translation
+export default function TeamExportModal({
+  show,
+  onHide,
+  teamId,
+}: {
+  show: boolean
+  onHide: () => void
+  teamId: string
+}) {
+  const database = useDatabase()
+  const team = useTeam(teamId)!
+  const { loadoutData } = team
```

Comment:
doesnt have to be this PR, but selecting multiple loadouts would be cool too. For example, sharing a Hutao + XQ team, where Hutao has 2 loadouts, one for shimenawa optimization, and one for non-shimenawa optimization (different opt target, set config, set filters, multi-opt, possibly ER constraints).

Similarly for XQ, there could be a Fav vs Sac loadout, which will have different multi-opt, and different ER constraint

Reply:
Seems kind of complicated. since this is on the team setting page, I think exiting the modal and selecting the team would be the way to go.

Reply:
but for this case, you go from a single export/import in your guide to 4 separate team exports/imports

shim + sac
shim + fav
no shim + sac
no shim + fav

singular import + swapping loadouts sounds simpler for a user than 2 import + team swap

Reply:
discussed in discord, should keep them as separate team exports for now, but is something we can think about if it becomes more common